app/models/user.rb

```diff
@@ -126,6 +126,10 @@ def create_reset_password_token
     raw # This value will be used to redirect users to the reset password page
  end
 
+  def employee_of?(company_id)
+    employments.pluck(:company_id).include?(company_id)
```

Comment:
```suggestion
    employments.exists?(company_id:)
```

This is a much cleaner implementation